profiles/arch/amd64/package.use.mask

```diff
@@ -479,7 +479,7 @@ media-libs/sdl-gfx cpu_flags_x86_mmx
 media-libs/smpeg cpu_flags_x86_mmx
 
 # x86_64 opts are enabled with USE sse. so masking the rest
-media-sound/mpg123 cpu_flags_x86_mmx cpu_flags_x86_3dnow cpu_flags_x86_3dnowext
```

Comment:
I think we need to duplicate this entry for the old mpg123 until new versions are stabled & old is cleaned up

Reply:
OK I restored original  masks in all 3 places